### ISSUE TITLE
Enhancement: Simultaneous creation of Memory and Hot Cues

### ISSUE THREAD

**Issue by andy-cam** [mganss/CueGen#10]
Hi Michael, this is an excellent tool - thank you for creating it!
I've spent the evening testing it with my rekordbox library though I wasn't able to get it to generate both Memory _and_ Hot Cues for my tracks.
(Why would one want to do this? Hot Cues are very handy on DJ Controllers, but older club gear and displays tend to only support Memory Cues - so personally I use them both the same way, with manually added memory cues where I want warnings etc).

Currently, if I have the tool generate one and then the other, they overwrite each other, even when using `--merge`.
Is this possible and I'm just holding it wrong?
Or if not, could it be implemented as an option?
Thanks in advance!

**Comment by mganss** [comment]
This is currently not supported. I think the easiest and most flexible way would be to require two passes for each type of cue point.

**Comment by mark-lawrenson** [comment]
Hi @mganss,

I have started using CueGen to create my hotcues from the phrases in Rekordbox. I can't believe none of this functionality is baked into rekordbox seeing as it is all there, but at least this solves the problem!!

I use hotcues to quickly preview a track and memory cues to count down bars before phrase changes, so also wanted to have identical memory and hot cues generated.

I saw your suggestion to run two passes, however it doesn't work because when running with or without `--hc` existing generated memory *and* hot cues are deleted. 

I've made a PR #21 which changes that behaviour and enables the two pass workflow to generate identical hot and memory cues.

Cheers
Mark
